src/Init/Data/Nat/Div.lean

```diff
@@ -39,6 +39,31 @@ theorem div.inductionOn.{u}
     base x y h
 decreasing_by apply div_rec_lemma; assumption
 
+theorem div_le_self (n k : Nat) : n / k ≤ n := by
+  induction n using Nat.strongInductionOn with
+  | ind n ih =>
+    rw [div_eq]
+    -- Note: manual split to avoid Classical.em which is not yet defined
+    cases (inferInstance : Decidable (0 < k ∧ k ≤ n)) with
```

Comment:
`split` fails here since `Classical.em` is not in scope, so I did this slightly strange manual decidable split.